packages/api/src/external/ehr/athenahealth/command/process-patients-from-appointments-sub.ts

```diff
@@ -1,103 +1,98 @@
-import dayjs from "dayjs";
-import duration from "dayjs/plugin/duration";
-import { errorToString } from "@metriport/shared";
-import { buildDayjs } from "@metriport/shared/common/date";
+import AthenaHealthApi, { AthenaEnv } from "@metriport/core/external/athenahealth/index";
 import { executeAsynchronously } from "@metriport/core/util/concurrency";
 import { out } from "@metriport/core/util/log";
 import { capture } from "@metriport/core/util/notifications";
-import AthenaHealthApi, { AthenaEnv } from "@metriport/core/external/athenahealth/index";
-import { EhrSources } from "../../shared";
+import { MetriportError, errorToString } from "@metriport/shared";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
 import { getCxMappingsBySource } from "../../../../command/mapping/cx";
-import { getSecretValueOrFail } from "@metriport/core/external/aws/secret-manager";
-import { Config } from "../../../../shared/config";
-import { getPatientIdOrFail as singleGetPatientIdOrFail } from "./get-patient";
+import { EhrSources, getLookackTimeRange } from "../../shared";
+import { getPatientIdOrFail } from "./get-patient";
+import { getAthenaEnv } from "../shared";
 
 dayjs.extend(duration);
 
 const delayBetweenPracticeBatches = dayjs.duration(30, "seconds");
-const lastModifiedHoursLookback = 12;
+const catupUpLookback = dayjs.duration(12, "hours");
 const parallelPractices = 10;
 const parallelPatients = 2;
 
-const region = Config.getAWSRegion();
-const athenaEnvironment = Config.getAthenaHealthEnv();
-const athenaClientKeySecretArn = Config.getAthenaHealthClientKeyArn();
-const athenaClientSecretSecretArn = Config.getAthenaHealthClientSecretArn();
-
 type PatientAppointment = {
   cxId: string;
   athenaPracticeId: string;
   athenaPatientId: string;
 };
 
-export async function getPatientIdsOrFailFromAppointmentsSub({
-  catchUp,
-}: {
-  catchUp: boolean;
-}): Promise<void> {
-  const { log } = out(`AthenaHealth getPatientIdsOrFailFromAppointmentsSub`);
-  if (!athenaEnvironment || !athenaClientKeySecretArn || !athenaClientSecretSecretArn) {
-    throw new Error("AthenaHealth not setup");
-  }
-  const cxMappings = await getCxMappingsBySource({ source: EhrSources.athena });
+export async function processPatientsFromAppointmentsSub({ catchUp }: { catchUp: boolean }) {
+  const { log } = out(`AthenaHealth getPatientIdsOrFailFromAppointmentsSub - catchUp: ${catchUp}`);
```

Comment:
nit: update the log string w/ the new function name

Reply:
Done.